packages/infra/src/constructs/lambdaDeployment.ts

```diff
@@ -61,5 +72,19 @@ export class LambdaDeployment extends Construct {
         'aws-codedeploy-hook-AfterAllowTraffic',
       ),
     );
+
+    if (!containsSkipDirective(buildMessage, 'alarm')) {
+      deploymentGroup.addAlarm(
+        new aws_cloudwatch.Alarm(this, 'codedeploy-alarm', {
+          metric: lambdaFunction.metricErrors({
+            period: Duration.seconds(30),
+          }),
+          threshold: 1,
+          evaluationPeriods: 1,
+          alarmDescription:
+            'Used to rollback the deployment if there are errors. This can be skipped with a [skip alarm] directive in the build message.',
```

Comment:
```suggestion
            'Used to roll back the deployment if there are errors. This can be skipped with a [skip alarm] directive in the build message.',
```